frontend/app/routes/$lang/_public/status/child.tsx

```diff
@@ -277,11 +277,18 @@ export default function StatusCheckerChild() {
       'first-name': errors?.firstName?._errors[0],
       'last-name': errors?.lastName?._errors[0],
       'date-picker-date-of-birth': errors?.dateOfBirth?._errors[0],
+      ...(i18n.language === 'fr'
```

Comment:
`errors?.dateOfBirth` is not mapped correctly... it should either be mapped to `date-picker-date-of-birth-day` when it's French or `date-picker-date-of-birth-month` when it's English

Reply:
`dateOfBirth` is not the same as `dateOfBirthDay`, it's for the whole date
French
```
'date-picker-date-of-birth-day': errors?.dateOfBirth?._errors[0],
```

English
```
'date-picker-date-of-birth-month': errors?.dateOfBirth?._errors[0],
```

Reply:
ran into some unexpected problems when updating, so I switched the implementation to how it's done in the Online Application.